parachains/launch-a-parachain/obtain-coretime.md

```diff
@@ -1,82 +1,177 @@
+tutorial_badge: Advanced
 categories: Parachains
 ---
 
 # Obtain Coretime
 
 ## Introduction
 
-Securing coretime is essential for operating a parachain on Polkadot. It provides your parachain with guaranteed computational resources and access to Polkadot's shared security model, ensuring your blockchain can process transactions, maintain its state, and interact securely with other parachains in the network. Without coretime, a parachain cannot participate in the ecosystem or leverage the relay chain's validator set for security.
+After deploying a parachain to Paseo in the [Deploy on Polkadot](/parachains/launch-a-parachain/deploy-on-polkadot/){target=\_blank} tutorial, the next critical step is obtaining coretime. Coretime is the mechanism through which validation resources are allocated from the relay chain to your parachain. Your parachain can only produce and finalize blocks on the relay chain by obtaining coretime.
 
-Coretime represents the computational resources allocated to your parachain on the Polkadot network. It determines when and how often your parachain can produce blocks and have them validated by the relay chain.
+There are two primary ways to obtain coretime:
 
-There are two primary methods to obtain coretime:
+- **[On-demand coretime](#order-on-demand-coretime)**: Purchase coretime on a block-by-block basis, ideal for variable or unpredictable workloads.
+- **[Bulk coretime](#purchase-bulk-coretime)**: Obtain a core or portion of a core for an extended period (up to 28 days), requiring renewal upon lease expiration.
 
-- **Bulk coretime**: Purchase computational resources in advance for a full month.
-- **On-demand coretime**: Buy computational resources as needed for individual block production.
+In this tutorial, you will:
 
-This guide explains the different methods of obtaining coretime and walks through the necessary steps to get your parachain running. 
+- Understand the different coretime options available.
+- Learn how to purchase a core via bulk coretime.
+- Assign your parachain to a core for block production.
+- Explore on-demand coretime as an alternative approach.
 
 ## Prerequisites
 
-Before obtaining coretime, ensure you have:
+Before proceeding, ensure you have the following:
 
-- Developed your parachain runtime using the Polkadot SDK.
-- Set up and configured a parachain collator for your target relay chain.
-- Successfully compiled your parachain collator node.
-- Generated and exported your parachain's genesis state.
-- Generated and exported your parachain's validation code (Wasm).
+- A parachain ID reserved on Paseo.
+- A properly configured chain specification file (both plain and raw versions).
+- A registered parathread with the correct genesis state and runtime.
+- A synced collator node running and connected to the Paseo relay chain.
+- PAS tokens in your account for transaction fees.
```

Comment:
@brunopgalvao are these supposed to be Paseo relay chain tokens? Like would this link be the correct one to provide? https://faucet.polkadot.io/?parachain=0

Reply:
They are supposed to be PAS tokens on the Coretime Chain.

The developer can use the Polkadot Faucet to get PAS tokens directly in their Paseo Coretime Chain account:
- https://faucet.polkadot.io/?parachain=1005

Instead of getting PAS tokens on Asset Hub  and teleporting it to the Coretime Chain.